index.html

```diff
@@ -64,7 +61,141 @@ <h2>LET’S CONNECT</h2>
                 </ul>
               </section>
             </section>
-          </section>
+        </div>
     </div>
+  </section>
+    <section class="deatails">
+      <div class="shot">
+      </div>
+      <div class="bo">
+      <div class="pro-title">
+        <p>Tonic</p>
+      </div>
+      <div class="item-parrent">
+        <div class="item1">
+          <p class="conopy">CONOPY</p>
+        </div>
+        <div class="item2"></div>
+        <div class="item3">
+          <p class="back">Back End Dev</p>
+        </div>
+        <div class="item4"></div>
+        <div class="item5">
+          <p class="year">2015</p></div>
+      </div>
+      <div class="text">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="tags">
+        <div class="html">
+          <p class="h">html</p>
+        </div>
+        <div class="css">
+          <p class="c">css</p>
+        </div>
+        <div class="js">
+          <p class="j">javascript</p>
+        </div>
+      </div>
+      <div class="action">
+        <div class="boxx">
+          <div class="see">
+            <p class="se">
+              See Project
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    </section>
+    <section class="footer">
+      <div class="top">
+        <div class="about">
+          <h1>About Myself</h1>
+        </div>
+      </div>
+      <div class="content">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </div>
+      <div class="footerbox">
+        <div class="let">
+          <h2 class="lets">LET’S CONNECT</h2>
+        </div>
+      </div>
+      <section class="social">
+        <ul class="social-icons-set">
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+          </li>
+          <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                aria-label="Medium"></i></a>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] As the previous reviewer suggested, kindly add a bit more padding or space between let's connect and social media icons to this section according to the Figma template. :+1: 

> * [ ]  Good job with the list of icons 1st_place_medal I think it would be better if you add a bit more `padding` or space to this section according to the Figma template. What do you think?

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/106119105/227724418-5f1b932b-2e18-4535-81d1-ce8044bd800f.png) | ![image](https://user-images.githubusercontent.com/106119105/227724442-f8ddcaf9-f316-487d-a3d6-3c854e98bc75.png)